src/explore-education-statistics-admin/src/components/editable/EditableKeyStatDataBlock.tsx

```diff
@@ -0,0 +1,113 @@
+import Button from '@common/components/Button';
+import ButtonGroup from '@common/components/ButtonGroup';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import WarningMessage from '@common/components/WarningMessage';
+import useToggle from '@common/hooks/useToggle';
+import useKeyStatQuery from '@common/modules/find-statistics/hooks/useKeyStatQuery';
+import React from 'react';
+import EditableKeyStatDataBlockForm from '@admin/components/editable/EditableKeyStatDataBlockForm';
+import { KeyStatisticDataBlock } from '@common/services/publicationService';
+import EditableKeyStatDisplay from '@admin/components/editable/EditableKeyStatDisplay';
+
+interface KeyStatsDataFormValues {
+  trend: string;
+  guidanceTitle: string;
+  guidanceText: string;
+}
+
+export interface EditableKeyStatDataBlockProps {
+  keyStat: KeyStatisticDataBlock;
+  isEditing?: boolean;
+  isReordering?: boolean;
+  onRemove?: () => void;
+  onSubmit: (values: KeyStatsDataFormValues) => void;
+  testId?: string;
+}
+
+const EditableKeyStatDataBlock = ({
+  keyStat: {
+    id: keyStatId,
+    releaseId,
+    dataBlockId,
+    trend,
+    guidanceTitle = 'Help',
+    guidanceText,
+  },
+  isEditing = false,
+  isReordering = false,
+  testId = 'keyStat',
+  onRemove,
+  onSubmit,
+}: EditableKeyStatDataBlockProps) => {
+  const [showForm, toggleShowForm] = useToggle(false);
+  const [removing, toggleRemoving] = useToggle(false);
+
+  const { value: dataBlockValues, isLoading, error } = useKeyStatQuery(
+    releaseId,
+    dataBlockId,
+  );
+
+  if (isLoading) {
+    return <LoadingSpinner />;
+  }
+
+  const fetchedTitle = dataBlockValues?.title;
```

Comment:
Very minor, but I'm not sure that `fetched` is a convention that we use in the front-end components at all - maybe there's a more conventional prefix that we use out there?

Reply:
I'm not sure. I'm happy to go with whatever you prefer.

Reply:
And I in turn will defer to Nick and Amy :)